config/index.ts

```diff
@@ -0,0 +1,10 @@
+const BASE_URLS = {
+  prod: 'https://api.id.gov.sg',
+  stag: 'https://api-stg.id.gov.sg',
+  dev: process.env.OVERRIDE_DEV || '',
+}
+const PORT = process.env.PORT || 10000
+const RANDOMNONCE = 'randomnonce'
+const SCOPES = process.env.SCOPES || 'openid'
+
+export { BASE_URLS, PORT, RANDOMNONCE, SCOPES }
```

Comment:
any reason why we use this syntax instead of just doing `export const` above? this syntax means we have to look in two places to check whether a function is exported

Reply:
Resolved in [73aca9f](https://github.com/opengovsg/sgid-test-app/pull/94/commits/73aca9fd057c093722ee6f95ba31f347607f4197).